torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9258,6 +9274,14 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                #                                          ~~~~~~ <--- HERE
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('bucketize',
+           dtypes=all_types_and(torch.bfloat16),
+           dtypesIfCUDA=all_types(),
+           sample_inputs_func=sample_inputs_bucketize,
+           supports_autograd=False,
+           skips=(
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```

Comment:
Add a comment for why this test is skipped

Reply:
Might as well make this an expected failure, too?

Reply:
How do I make something an expected failure? I couldn't find an example elsewhere in OpInfos.